docs/usage.md

```diff
@@ -13,6 +13,30 @@ General process:
 4. set the environment variables ([Containerfile example](#write-the-dockerfile-or-containerfile))
 5. run the build ([container build example](#build-the-container))
 
+## Configuration
+
+There are several parameters with default values set. See the [config.py](cachi2/core/config.py) module for defaults.
```

Comment:
The link is a 404: https://github.com/ejegrova/cachi2/blob/config_docs/docs/cachi2/core/config.py

I think it needs to be `../cachi2/core/config.py`